build-recipe-preinstallimage

```diff
@@ -70,7 +81,7 @@ recipe_build_preinstallimage() {
 	test -n "$PKG_HDRMD5" || cleanup_and_exit 1
 	echo "$PKG_HDRMD5  $PKG" >> $BUILD_ROOT$TOPDIR/OTHER/preinstallimage.info
     done
-    mv $BUILD_ROOT/.preinstallimage.$$.tar.gz $BUILD_ROOT$TOPDIR/OTHER/preinstallimage.tar.gz
+    mv $BUILD_ROOT/.preinstallimage.$$.tar.$ext $BUILD_ROOT$TOPDIR/OTHER/preinstallimage.tar.$ext || cleanup_and_exit 1
```

Comment:
No need for $ext, just use `*`

Reply:
that doesn't work because for some reason that I don't understand it is being renamed from `.preinstallinstallimage.$$.tar.$ext` to `preinstallimage.tar.$ext` with this move.

Reply:
seems like it has been there from day one. removed as there shouldn't be concurrent runs as far as I can see possible..

Reply:
its gone now.